.idea/workspace.xml

```diff
@@ -1,45 +1,83 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All of your .idea files can be added to .gitignore

Reply:
Ok will do so and make another pull request 

But it may take some time since I am not home